config/evaluation_data.yaml

```
@@ -0,0 +1,91 @@
+# LightSpeed Evaluation Framework - Sample/Mock Data
+
+- conversation_group_id: "conv_group_1"
+  description: "conversation group description"
+
+  turn_metrics:
+    - "ragas:faithfulness"
+    - "ragas:response_relevancy"
+    - "ragas:context_precision_without_reference"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)